src/macros.jl

```diff
@@ -128,6 +128,17 @@ struct TestItem
     stats::Vector{PerfStats} # populated when the test item is finished evaluating
     scheduled_for_evaluation::ScheduledForEvaluation # to keep track of whether the test item has been scheduled for evaluation
 end
+function TestItem(id, name, tags, default_imports, setups, retries, file, line, project_root, code)
+    return TestItem(
+        id, name, tags, default_imports, setups, retries, file, line, project_root, code,
+        TestSetup[],
+        Ref{Int}(0),
+        DefaultTestSet[],
+        Ref{Int}(0),
+        PerfStats[],
+        ScheduledForEvaluation(),
+    )
```

Comment:
since these fields are always set the same, i thought it made sense to have a simpler constructor; it makes manually constructing a `TestItem` in the tests a lot easier